tests/test_compute_funs.py

```diff
@@ -413,3 +416,190 @@ def test_surface_areas():
     np.testing.assert_allclose(Ar, 4 * 10 * np.pi ** 2)
     np.testing.assert_allclose(At, np.pi * (11 ** 2 - 10 ** 2))
     np.testing.assert_allclose(Az, np.pi)
+
+
+def test_vector_signs():
+
+    R_lmn = np.array([10, 1, 0.2])
+    modes_R = np.array([[0, 0], [1, 0], [-1, 1]])
+    Z_lmn = np.array([0, -2, -0.2])
+    modes_Z = np.array([[0, 0], [-1, 0], [-1, 1]])
+    surfacep = FourierRZToroidalSurface(R_lmn, Z_lmn, modes_R, modes_Z, NFP=1)
+    R_lmn = np.array([10, 1, -0.2])
+    modes_R = np.array([[0, 0], [1, 0], [-1, 1]])
+    Z_lmn = np.array([0, 2, 0.2])
+    modes_Z = np.array([[0, 0], [-1, 0], [-1, 1]])
+    surfacem = FourierRZToroidalSurface(R_lmn, Z_lmn, modes_R, modes_Z, NFP=1)
+
+    iotap = PowerSeriesProfile([1, 0, -0.5])
+    iotam = PowerSeriesProfile([-1, 0, 0.5])
+
+    # ppp = jacobian plus, psi plus, iota plus
+    # mmp = jacobian minus, psi minus, iota plus
+    # etc.
+
+    eqmmm = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=-1
+    )
+    eqmmp = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=-1
+    )
+    eqmpm = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=1
+    )
+    eqmpp = Equilibrium(
+        surface=surfacem, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=1
+    )
+    eqpmm = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=-1
+    )
+    eqpmp = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=-1
+    )
+    eqppm = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotam, Psi=1
+    )
+
+    eqppp = Equilibrium(
+        surface=surfacep, L=5, M=10, N=5, NFP=1, sym=False, iota=iotap, Psi=1
+    )
+
+    grid = Grid(np.array([[1, 0, 0]]))
+
+    # jacobian sign
+    assert np.sign(eqmmm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmmp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmpm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqmpp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == -1.0
+    assert np.sign(eqpmm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqpmp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqppm.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+    assert np.sign(eqppp.compute("sqrt(g)", grid=grid)["sqrt(g)"][0]) == 1.0
+
+    grid = Grid(np.array([[0.1, 0, 0]]))
```

Comment:
Why can't the toroidal field and current be evaluated at $\rho=1$ like the other quantities?